examples/test.py

```diff
@@ -0,0 +1,8 @@
+"""
```

Comment:
This looks unfinished, too.

Reply:
Also needs to be removed, I think this is an artifact of setting up the sphinx-automl-theme

Reply:
We could also move the user guide to the examples? Like here https://automl.github.io/automl_sphinx_theme/main/examples/index.html

Reply:
If you're willing to move each actual examples in the user guide to their own examples then go for it, otherwise I would advocate to just delete the need for examples. Would this break the theme?